docs/en/docs/deployment/docker.md

```diff
@@ -206,7 +206,7 @@ CMD ["uvicorn", "app.main:app", "--host", "0.0.0.0", "--port", "80"]
 
     Because the previous step copying the file could be detected by the **Docker cache**, this step will also **use the Docker cache** when available.
 
-    Using the cache in this step will **save** you a lot of **time** when building the image again and again during development, instead of **downloading and installing** all the dependencies **every time**.
+    Using the cache in this step will **save** your lot of **time** when building the image again and again during development, instead of **downloading and installing** all the dependencies **every time**.
```

Comment:
old text was correct, new is wrong.
the intention was "it will save you time", this "this step will save you a lot of time".